.gitleaks.toml

```diff
@@ -534,8 +534,12 @@ regex = '''(?i)((key|api|token|secret|password)[a-z0-9_ .\-,]{0,25})(=|>|:=|\|\|
 entropy = 3.7
 secretGroup = 4
 
-
 [allowlist]
-description = "global allow lists"
-regexes = ['''219-09-9999''', '''078-05-1120''', '''(9[0-9]{2}|666)-\d{2}-\d{4}''']
-paths = ['''(.*?)(jpg|gif|doc|pdf|bin|svg|socket)$''']
+description = "allowlist"
+regexTarget = "match"
+regexes = [
+    '''\b(0x)?[0-9a-fA-F]{40}\b''', # ignore evm public keys
+    '''219-09-9999''', # global allow lists
+    '''078-05-1120''',
+    '''(9[0-9]{2}|666)-\d{2}-\d{4}'''
```

Comment:
i'm not sure what these are, they're probably defaults from elsewhere?
can I remove these?